cluster.go

```diff
@@ -422,14 +422,22 @@ func (c *Cluster) run() {
 	go c.alertsHandler()
 }
 
-func (c *Cluster) ready() {
+func (c *Cluster) ready(timeout time.Duration) {
 	// We bootstrapped first because with dirty state consensus
 	// may have a peerset and not find a leader so we cannot wait
 	// for it.
-	timer := time.NewTimer(30 * time.Second)
+	timer := time.NewTimer(timeout)
 	select {
 	case <-timer.C:
-		logger.Error("consensus start timed out")
+		logger.Error("**************************************************")
```

Comment:
Could this be one `logger.Error` call with the entire message body being in a string literal?

Reply:
It can, but then lines are not prefixed with ERROR, which might hide them if your log processing is grepping for ERROR lines. Hmm but looks definitely prettier, so I think I'm going to partially do it.